docs/libs/table.md

```diff
@@ -123,13 +123,205 @@ DataSet<WC> result = tableEnv.toDataSet(filtered, WC.class);
 When using Java, the embedded DSL for specifying expressions cannot be used. Only String expressions
 are supported. They support exactly the same feature set as the expression DSL.
 
-## Expression Syntax
+## Table API Operators
+Table API provide a domain-spcific language to execute language-integrated query on structured data in Scala and Java.
+This section gives a brief overview of all available operators. You can find more details of operators in the [Javadoc](http://flink.apache.org/docs/latest/api/java/org/apache/flink/api/table/Table.html).
```

Comment:
Where should i link to? The javdoc link of 1.0 version does not exist yet, should i link to 1.0 here anyway, or link to 0.9 here, and let release manager update to 1.0 while release 1.0?